CS has been splitted into ME and RS.
Upstream already added sr_ME into CVS.
Keep sr_CS for backward compatibility, and add a new sr_RS
locale which copies definitions from sr_CS.  In the future,
sr_CS will be renamed into sr_RS.

Index: localedata/SUPPORTED
===================================================================
--- localedata/SUPPORTED.orig
+++ localedata/SUPPORTED
@@ -336,6 +336,7 @@
 sr_CS.UTF-8/UTF-8 \
 sr_CS/ISO-8859-5 \
 sr_ME/UTF-8 \
+sr_RS/UTF-8 \
 ss_ZA/UTF-8 \
 st_ZA.UTF-8/UTF-8 \
 st_ZA/ISO-8859-1 \
Index: localedata/locales/sr_RS
===================================================================
--- /dev/null
+++ localedata/locales/sr_RS
@@ -0,0 +1,95 @@
+comment_char %
+escape_char  /
+%
+% Serbian Language Locale for Serbia
+% Source: sr_YU locale
+% Address:
+%
+% Contact: Danilo Segan
+% Email: dsegan@gmx.net
+% Tel: +381 63 8626082
+% Fax: none
+% Language: sr
+% Territory: RS
+% Revision: 1.0
+% Date: 2003-08-13
+% Application: general
+% Users: general
+% Charset: UTF-8, ISO-8859-5
+% Distribution and use is free, also
+% for commercial purposes.
+%
+
+LC_IDENTIFICATION
+title      "Serbian locale for Serbia"
+source     "sr_YU locale"
+address    ""
+contact    "Danilo Segan"
+email      "bug-glibc@gnu.org"
+tel        ""
+fax        ""
+language   "Serbian"
+territory  "Serbia"
+revision   "1.1"
+date       "2004-01-10"
+audience   "general"
+application   ""
+abbreviation  ""
+%
+category  i18n:1997;LC_IDENTIFICATION
+category  i18n:1997;LC_CTYPE
+category  i18n:1997;LC_COLLATE
+category  i18n:1997;LC_MONETARY
+category  i18n:1997;LC_NUMERIC
+category  i18n:1997;LC_TIME
+category  i18n:1997;LC_MESSAGES
+category  i18n:1997;LC_PAPER
+category  i18n:1997;LC_NAME
+category  i18n:1997;LC_ADDRESS
+category  i18n:1997;LC_TELEPHONE
+category  i18n:1997;LC_MEASUREMENT
+END LC_IDENTIFICATION
+
+LC_CTYPE
+copy "sr_CS"
+END LC_CTYPE
+
+LC_COLLATE
+copy "sr_CS"
+END LC_COLLATE
+
+LC_MONETARY
+copy "sr_CS"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "sr_CS"
+END LC_NUMERIC
+
+LC_TIME
+copy "sr_CS"
+END LC_TIME
+
+LC_MESSAGES
+copy "sr_CS"
+END LC_MESSAGES
+
+LC_PAPER
+copy "sr_CS"
+END LC_PAPER
+
+LC_NAME
+copy "sr_CS"
+END LC_NAME
+
+LC_ADDRESS
+copy "sr_CS"
+END LC_ADDRESS
+
+LC_TELEPHONE
+copy "sr_CS"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "sr_CS"
+END LC_MEASUREMENT
